### ISSUE TITLE
What does the inv_temp variable mean?

### ISSUE THREAD

**Issue by hnyll** [Parskatt/DeDoDe#26]
i was looking at the code from

https://github.com/Parskatt/DeDoDe/blob/main/demo/demo_match.py

and at line 42 there is a variable named inv_temp  but there isn't much description for what it is.
Can someone clarify?

**Comment by Parskatt** [comment]
We match with softmax over scaled cosine similarly. We call the scaling inverse temperature. People often refer to temperature when scaling logits.
